packages/contentstack-export/src/interfaces/index.ts

```diff
@@ -0,0 +1,25 @@
+// eslint-disable-next-line @typescript-eslint/no-redeclare
+export interface AuthOptions {
+  contentstackClient: any;
+}
+
+export interface ContentStackManagementClient {
+  contentstackClient: object;
+}
+
+export interface PrintOptions {
+  color?: string;
+}
+
+export interface InquirePayload {
```

Comment:
@shafeeqd959 Can you recheck the InquirePayload types from their package? One of my observation is **_validate_** option is missing from the list I could see.